docs/docs/applications/add-application.md

```diff
@@ -5,7 +5,7 @@ links:
     overview:
     quickstart:
     previous: servers/ssh-into-server
-    next: pipelines/steps/add-step
+    next: applications/list-applications
```

Comment:
Should we really take the reader of the documentation to learn how to list application s after already have added applications?

Remember: the documentation is to help somebody achieve some task and move forward. If the web app UI already taskes user through the list before adding an application, the documentation might not need to explain everything that is already implemented in the UI.

See similar discussions:
1. https://github.com/devopness/devopness/pull/1915#discussion_r2174578366
2. https://github.com/devopness/devopness/pull/1904#discussion_r2169835299